Update kentico.gitignore

cjfphilip opened the pull request:
I noticed that when installing Kentico 10 the DeviceProfiles folder is plural and not singular "DeviceProfile". Unless this ignore file is for a different version of Kentico.

cjfphilip committed:
Update kentico.gitignore

I noticed that when installing Kentico 10 the DeviceProfiles folder is plural and not singular "DeviceProfile". Unless this ignore file is for a different version of Kentico.

ChristopherJennings commented on kentico.gitignore:
```diff
@@ -270,7 +270,7 @@ $RECYCLE.BIN/
 CMS/App_Data/AzureCache
 CMS/App_Data/AzureTemp
-CMS/App_Data/CMSModules/DeviceProfile/logFiftyOne.txt
+CMS/App_Data/CMSModules/DeviceProfiles/logFiftyOne.txt
```
Can you modify this to simply add another line ignoring the plural version for version 10?

ChristopherJennings reviewed (changes requested):
Hi Charles, 

Thanks for the pull request. This ignore was originally designed for Kentico version 9. Rather than breaking the ignore for 9, can you modify this to simply add another line ignoring the plural version for version 10? Have you found anything else that needs to be re-ignored, added or removed?

cjfphilip committed:
Update kentico.gitignore

I wish you could just add (s) to and not have to add an extra line, but that doesn't seem to work

cjfphilip commented:
I have added this to my local file:
```
**/[Bb]in/**/*
!**/[Bb]in/**/
!*.refresh
```
so that .refresh files will be included to restore missing dll's. but I'm not sure if this is good or not, so I haven't submitted it in here.

ChristopherJennings commented:
Thanks for the update! regarding the secondary changes you mentioned, those are file-based references specific to web site projects (rather than web application projects). My understanding is that they should be included in source control as they point to where the project can get the DLL if needed. It wouldn't be a bad idea to include this in the ignore, but commented out with an explanation as to why you would uncomment it. If you'd like to contribute this, please open another pull request, and I'll happily bring that in.
